### ISSUE TITLE
Invalid json return on API ldap/users/import

### ISSUE THREAD

**Issue by vasartori** [vmware/harbor#4901]
On Harbor 1.5 when we try import users from ldap by api (/api/ldap/users/import) the return of unknown users returns a invalid json. The problem is on first line with a text `Import LDAP user is not found`.
See the return:

```
Import LDAP user is not found
[
  {
    "uid": "xxxxxxxxx",
    "err_msg": "unknown_user"
  },
  {
    "uid": "xxxxxxxx",
    "err_msg": "unknown_user"
  },
  {
    "uid": "xxxxxxx",
    "err_msg": "unknown_user"
  }
]
```

**Comment by stonezdj** [comment]
Fixed in #4925

**Comment by reasonerjt** [comment]
@vasartori thanks for reporting this issue.

Please be kindly reminded that this API is not stable and we may deprecate it in future.

Closing as the fix is merged.
